.github/workflows/cli-tests.yml

```diff
@@ -69,8 +76,10 @@ jobs:
           browser: chrome
           project: ./packages/e2e
           spec: packages/e2e/cypress/cli/**/*
-          config: 'baseUrl=${{steps.deployment_url.outputs.url}}'
+          config: 'baseUrl=${{needs.prepare-preview.outputs.url}}'
         env:
           CYPRESS_PGHOST: ${{secrets.PGHOST}}
           CYPRESS_PGPASSWORD: ${{secrets.PGPASSWORD}}
+          SEED_SCHEMA: jaffle-${{strategy.job-index}}
```

Comment:
I think this should be CYPRESS_SEED_SCHEMA if you want to get this using `Cypress.env('SEED_SCHEMA')`